string port
int16 min_pwm0
int16 min_pwm1
int16 max_pwm0
int16 max_pwm1
---
bool success
